cli/flox-rust-sdk/src/models/environment/core_environment.rs

```diff
@@ -263,7 +264,8 @@ impl<State> CoreEnvironment<State> {
 
         let service_config_path = maybe_make_service_config_file(flox, &lockfile)?;
 
-        let outputs = BuildEnvNix::new("/tmp/flox-temp-roots/").build(
+        let tempdir = TempDir::new().map_err(CoreEnvironmentError::CreateTempdir)?;
```

Comment:
FWIW I think this was a change I accidentally committed 
I originally created a temproots in the process directory which would get deleted upon the end of the flox process.


As matthew noted, these process does are kept around when you run flox with -v which would be bad for gc roots.
I said we come back to this and apparently committed this one still.